source/integrations.txt

```diff
@@ -0,0 +1,142 @@
+.. _java-integrations:
+
+===================
+Driver Integrations
+===================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: java sync, third party, tools, frameworks
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+On this page, you can learn about using tools and frameworks together
+with the {+driver-short+} to develop applications.
+
+This page describes the following third-party integrations:
+
+- :ref:`spring-framework`
+- :ref:`quarkus-framework`
+- :ref:`micronaut-framework`
+
+.. TODO (add after JAVA-3580 is finished):
+   - :ref:`graalvm-integration`
+
+.. important::
+
+   The integrations on this page are not official MongoDB partners.
+
+.. _spring-framework:
+
+Spring Data
+-----------
+
+`Spring <https://spring.io/>`__ is a Java framework that provides infrastructure support
+for Java applications. Spring's modules enable aspect-oriented programming, authentication
+protocols, inversion of control, and other services to facilitate application development.
+
+Spring offers the Spring Data sub-framework for database access. Spring Data's features include
+map-reduce frameworks, cloud-based data services, and an access layer for relational and
+non-relational databases. To learn more about the Spring Data project, see `Spring Data
+<https://spring.io/projects/spring-data>`__.
+
+Using Spring Data and MongoDB
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Spring Data MongoDB enables you to access MongoDB databases by using a Spring-based programming
+model. 
+
+You can use Spring Data MongoDB's template API to perform the following actions:
```

Comment:
Spring Data supports a bunch of MongoDB capabilities outside of CRUD. Could we call out that Atlas Search is supported, as well as CSFLE?